spec/ttml3.xml

```diff
@@ -1410,6 +1434,22 @@ consist of one or more <loc href="#terms-inline-area">inline areas</loc>.</p>
 <p>Any of the element types defined by the <loc href="#element-vocab-type-metadata">Metadata Module</loc>.</p>
 </def>
 </gitem>
+<gitem id="terms-module">
+<label>[module]</label>
+<def>
+<p>A collection of element types, attributes, features, or the specification(s) of such entities, where the definition of a module
```

Comment:
> or the specification(s) of such entities, where the definition of a module and the context of a reference to a module determine the nature of the defined or referenced entities

What is this end of sentence adding? Can we get rid of it? What is an 'entity' in this case?

Reply:
I think this definition of module is too specific. I would rather merely define it as a separate document that defines a part of the specification.

Reply:
@cconcolato deleted that portion of text in https://github.com/w3c/ttml3/pull/30/commits/eeb4d4e77e42a291ff17c711651d396cde457cff.